source/reference/methods.txt

```diff
@@ -876,3 +876,90 @@ Role Management Methods
    * - :method:`db.updateRole()`
 
      - Updates a user-defined role.
+
+Replication Methods 
+-------------------
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Method
+
+     - Description
+ 
+   * - ``rs.printSecondaryReplicationInfo()``
+
+     - Returns the status of the secondary members of the replica set. This 
```

Comment:
Suggest putting a linebreak after the first sentence so the description of the method stands on its own and is easy to spot.